modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullScreenPluginTest.ts

```diff
@@ -73,6 +75,14 @@ describe('browser.tinymce.plugins.fullscreen.FullScreenPluginTest', () => {
     assertShadowHostState(editor, shouldExist);
   };
 
+  const keystokes = (editor: Editor) => {
+    if (platform.os.isMacOS()) {
+      TinyContentActions.keystroke(editor, 'F'.charCodeAt(0), { meta: true, shift: true });
+    } else {
+      TinyContentActions.keystroke(editor, 'F'.charCodeAt(0), { ctrl: true, shift: true });
```

Comment:
This has more duplication that what we need, so it should be shortened to something like this:
```suggestion
    const modifiers = platform.os.isMacOS() ? { meta: true, shift: true } :  { ctrl: true, shift: true };
    TinyContentActions.keystroke(editor, 'F'.charCodeAt(0), modifiers);
```